packages/style-engine/class-wp-style-engine.php

```diff
@@ -211,26 +309,27 @@ protected static function get_css( $style_value, $style_definition, $should_retu
 				foreach ( $style_definition['css_vars'] as $css_var_pattern => $property_key ) {
 					$slug = static::get_slug_from_preset_value( $style_value, $property_key );
 					if ( $slug ) {
-						$css_var                  = strtr(
+						$css_var                               = strtr(
 							$css_var_pattern,
 							array( '$slug' => $slug )
 						);
-						$rules[ $style_property ] = "var($css_var)";
+						$rules[ $style_properties['default'] ] = "var($css_var)";
 					}
 				}
 			}
 			return $rules;
 		}
 
 		// Default rule builder.
-		// If the input contains an array, assume box model-like properties
+		// If the input contains an array, we assume box model-like properties
 		// for styles such as margins and padding.
 		if ( is_array( $style_value ) ) {
 			foreach ( $style_value as $key => $value ) {
-				$rules[ "$style_property-$key" ] = $value;
+				$side_property           = sprintf( $style_properties['individual'], _wp_to_kebab_case( $key ) );
+				$rules[ $side_property ] = $value;
```

Comment:
```suggestion
				$individual_property           = sprintf( $style_properties['individual'], _wp_to_kebab_case( $key ) );
				$rules[ $individual_property ] = $value;
```

The plan is to replace all the "side" based terminology right?